src/scene_equip.cpp

```diff
@@ -126,6 +131,11 @@ void Scene_Equip::UpdateStatusWindow() {
 		spi = Utils::Clamp(spi, 1, 999);
 		agi = Utils::Clamp(agi, 1, 999);
 
+		atk = actor.CalcValueAfterStates(atk, &lcf::rpg::State::affect_attack);
+		def = actor.CalcValueAfterStates(def, &lcf::rpg::State::affect_defense);
+		spi = actor.CalcValueAfterStates(spi, &lcf::rpg::State::affect_spirit);
+		agi = actor.CalcValueAfterStates(agi, &lcf::rpg::State::affect_agility);
```

Comment:
We should add wrapper functions instead of requiring the caller to pass a pointer to member.